<!DOCTYPE html>
<html>

<head>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<title>profiles_objectives_special_comps</title>


<style type="text/css">
body {
  font-family: Helvetica, arial, sans-serif;
  font-size: 14px;
  line-height: 1.6;
  padding-top: 10px;
  padding-bottom: 10px;
  background-color: white;
  padding: 30px; }

body > *:first-child {
  margin-top: 0 !important; }
body > *:last-child {
  margin-bottom: 0 !important; }

a {
  color: #4183C4; }
a.absent {
  color: #cc0000; }
a.anchor {
  display: block;
  padding-left: 30px;
  margin-left: -30px;
  cursor: pointer;
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0; }

h1, h2, h3, h4, h5, h6 {
  margin: 20px 0 10px;
  padding: 0;
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  cursor: text;
  position: relative; }

h1:hover a.anchor, h2:hover a.anchor, h3:hover a.anchor, h4:hover a.anchor, h5:hover a.anchor, h6:hover a.anchor {
  background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA09pVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMy1jMDExIDY2LjE0NTY2MSwgMjAxMi8wMi8wNi0xNDo1NjoyNyAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNiAoMTMuMCAyMDEyMDMwNS5tLjQxNSAyMDEyLzAzLzA1OjIxOjAwOjAwKSAgKE1hY2ludG9zaCkiIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6OUM2NjlDQjI4ODBGMTFFMTg1ODlEODNERDJBRjUwQTQiIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6OUM2NjlDQjM4ODBGMTFFMTg1ODlEODNERDJBRjUwQTQiPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDo5QzY2OUNCMDg4MEYxMUUxODU4OUQ4M0REMkFGNTBBNCIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDo5QzY2OUNCMTg4MEYxMUUxODU4OUQ4M0REMkFGNTBBNCIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/PsQhXeAAAABfSURBVHjaYvz//z8DJYCRUgMYQAbAMBQIAvEqkBQWXI6sHqwHiwG70TTBxGaiWwjCTGgOUgJiF1J8wMRAIUA34B4Q76HUBelAfJYSA0CuMIEaRP8wGIkGMA54bgQIMACAmkXJi0hKJQAAAABJRU5ErkJggg==) no-repeat 10px center;
  text-decoration: none; }

h1 tt, h1 code {
  font-size: inherit; }

h2 tt, h2 code {
  font-size: inherit; }

h3 tt, h3 code {
  font-size: inherit; }

h4 tt, h4 code {
  font-size: inherit; }

h5 tt, h5 code {
  font-size: inherit; }

h6 tt, h6 code {
  font-size: inherit; }

h1 {
  font-size: 28px;
  color: black; }

h2 {
  font-size: 24px;
  border-bottom: 1px solid #cccccc;
  color: black; }

h3 {
  font-size: 18px; }

h4 {
  font-size: 16px; }

h5 {
  font-size: 14px; }

h6 {
  color: #777777;
  font-size: 14px; }

p, blockquote, ul, ol, dl, li, table, pre {
  margin: 15px 0; }

hr {
  background: transparent url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAYAAAAECAYAAACtBE5DAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyJpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNSBNYWNpbnRvc2giIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6OENDRjNBN0E2NTZBMTFFMEI3QjRBODM4NzJDMjlGNDgiIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6OENDRjNBN0I2NTZBMTFFMEI3QjRBODM4NzJDMjlGNDgiPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDo4Q0NGM0E3ODY1NkExMUUwQjdCNEE4Mzg3MkMyOUY0OCIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDo4Q0NGM0E3OTY1NkExMUUwQjdCNEE4Mzg3MkMyOUY0OCIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/PqqezsUAAAAfSURBVHjaYmRABcYwBiM2QSA4y4hNEKYDQxAEAAIMAHNGAzhkPOlYAAAAAElFTkSuQmCC) repeat-x 0 0;
  border: 0 none;
  color: #cccccc;
  height: 4px;
  padding: 0;
}

body > h2:first-child {
  margin-top: 0;
  padding-top: 0; }
body > h1:first-child {
  margin-top: 0;
  padding-top: 0; }
  body > h1:first-child + h2 {
    margin-top: 0;
    padding-top: 0; }
body > h3:first-child, body > h4:first-child, body > h5:first-child, body > h6:first-child {
  margin-top: 0;
  padding-top: 0; }

a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 {
  margin-top: 0;
  padding-top: 0; }

h1 p, h2 p, h3 p, h4 p, h5 p, h6 p {
  margin-top: 0; }

li p.first {
  display: inline-block; }
li {
  margin: 0; }
ul, ol {
  padding-left: 30px; }

ul :first-child, ol :first-child {
  margin-top: 0; }

dl {
  padding: 0; }
  dl dt {
    font-size: 14px;
    font-weight: bold;
    font-style: italic;
    padding: 0;
    margin: 15px 0 5px; }
    dl dt:first-child {
      padding: 0; }
    dl dt > :first-child {
      margin-top: 0; }
    dl dt > :last-child {
      margin-bottom: 0; }
  dl dd {
    margin: 0 0 15px;
    padding: 0 15px; }
    dl dd > :first-child {
      margin-top: 0; }
    dl dd > :last-child {
      margin-bottom: 0; }

blockquote {
  border-left: 4px solid #dddddd;
  padding: 0 15px;
  color: #777777; }
  blockquote > :first-child {
    margin-top: 0; }
  blockquote > :last-child {
    margin-bottom: 0; }

table {
  padding: 0;border-collapse: collapse; }
  table tr {
    border-top: 1px solid #cccccc;
    background-color: white;
    margin: 0;
    padding: 0; }
    table tr:nth-child(2n) {
      background-color: #f8f8f8; }
    table tr th {
      font-weight: bold;
      border: 1px solid #cccccc;
      margin: 0;
      padding: 6px 13px; }
    table tr td {
      border: 1px solid #cccccc;
      margin: 0;
      padding: 6px 13px; }
    table tr th :first-child, table tr td :first-child {
      margin-top: 0; }
    table tr th :last-child, table tr td :last-child {
      margin-bottom: 0; }

img {
  max-width: 100%; }

span.frame {
  display: block;
  overflow: hidden; }
  span.frame > span {
    border: 1px solid #dddddd;
    display: block;
    float: left;
    overflow: hidden;
    margin: 13px 0 0;
    padding: 7px;
    width: auto; }
  span.frame span img {
    display: block;
    float: left; }
  span.frame span span {
    clear: both;
    color: #333333;
    display: block;
    padding: 5px 0 0; }
span.align-center {
  display: block;
  overflow: hidden;
  clear: both; }
  span.align-center > span {
    display: block;
    overflow: hidden;
    margin: 13px auto 0;
    text-align: center; }
  span.align-center span img {
    margin: 0 auto;
    text-align: center; }
span.align-right {
  display: block;
  overflow: hidden;
  clear: both; }
  span.align-right > span {
    display: block;
    overflow: hidden;
    margin: 13px 0 0;
    text-align: right; }
  span.align-right span img {
    margin: 0;
    text-align: right; }
span.float-left {
  display: block;
  margin-right: 13px;
  overflow: hidden;
  float: left; }
  span.float-left span {
    margin: 13px 0 0; }
span.float-right {
  display: block;
  margin-left: 13px;
  overflow: hidden;
  float: right; }
  span.float-right > span {
    display: block;
    overflow: hidden;
    margin: 13px auto 0;
    text-align: right; }

code, tt {
  margin: 0 2px;
  padding: 0 5px;
  white-space: nowrap;
  border: 1px solid #eaeaea;
  background-color: #f8f8f8;
  border-radius: 3px; }

pre code {
  margin: 0;
  padding: 0;
  white-space: pre;
  border: none;
  background: transparent; }

.highlight pre {
  background-color: #f8f8f8;
  border: 1px solid #cccccc;
  font-size: 13px;
  line-height: 19px;
  overflow: auto;
  padding: 6px 10px;
  border-radius: 3px; }

pre {
  background-color: #f8f8f8;
  border: 1px solid #cccccc;
  font-size: 13px;
  line-height: 19px;
  overflow: auto;
  padding: 6px 10px;
  border-radius: 3px; }
  pre code, pre tt {
    background-color: transparent;
    border: none; }

sup {
    font-size: 0.83em;
    vertical-align: super;
    line-height: 0;
}

kbd {
  display: inline-block;
  padding: 3px 5px;
  font-size: 11px;
  line-height: 10px;
  color: #555;
  vertical-align: middle;
  background-color: #fcfcfc;
  border: solid 1px #ccc;
  border-bottom-color: #bbb;
  border-radius: 3px;
  box-shadow: inset 0 -1px 0 #bbb
}

* {
	-webkit-print-color-adjust: exact;
}
@media screen and (min-width: 914px) {
    body {
        width: 854px;
        margin:0 auto;
    }
}
@media print {
	table, pre {
		page-break-inside: avoid;
	}
	pre {
		word-wrap: break-word;
	}
  body {
    padding: 2cm; 
  }
}
</style>


</head>

<body>

<h2 id="toc_0">Profile</h2>

<h3 id="toc_1">general</h3>

<p>Expert in applied bioinformatics with 8+ years of experience in providing transparent and reproducible solutions for diverse areas of biological and medical research. Deep knowledge of underlying issues in need of bioinformatics support. Experience includes genetic epidemiology of rare diseases, integrative omics analysis of data from cancer tissue and immune cells, microarray analysis, preprocessing to final analysis of NGS output, sequence alignment and assembly working with non-human genomes from bacteria to fish species and wolf in evolutionary research and finally integrative analysis of proteomics, clinical and brain imaging data. Experience running jobs with queuing systems on high performance clusters. Focused on keeping at the forefront of the latest within computational omics analysis. I seek to understand the healthcare environment and external trends. Emotional intelligence, strong communication, great teammate. My collegues say about me that I am hardworking, joyful and problemsolving.</p>

<h3 id="toc_2">2022 04 30 Clinical Application Consultant NGS themo fisher</h3>

<p>Expert in applied bioinformatics with 8+ years of experience in providing transparent and reproducible solutions for diverse areas of biological and medical research. Deep knowledge of underlying issues in need of bioinformatics support. Experience includes genetic epidemiology of rare diseases, integrative omics analysis of data from cancer tissue and immune cells, microarray analysis, preprocessing to final analysis of NGS (Illumina and Pacbio) output, sequence alignment and assembly working with non-human genomes from bacteria to fish species and wolf in evolutionary research and finally integrative analysis of proteomics, clinical and brain imaging data. I am used to work in a service position listening to the needs of collborators or customers and come up with suggestions from a bioinformatics perspective for the best implementation. Focused on keeping at the forefront of the latest within computational omics analysis. I seek to understand the healthcare environment and external trends. Emotional intelligence, strong communication, great teammate. My collegues say about me that I am hardworking, joyful, problemsolving and serviceminded.</p>

<h3 id="toc_3">2022 04 17 Sen Sci Mol Bio Bioinf AZ</h3>

<p>Expert in applied bioinformatics with 8+ years of experience in providing transparent and reproducible solutions for diverse areas of biological and medical research. I am innovative and have general drive to to find solutions to technological problems. In my thesis project for MS in molecular biology I worked with cloning of a tumor supressor gene. I seek to understand the healthcare environment and external trends. I have emotional intelligence, strong communication skills and am a great teammate. I am dynamic and work very hard to achieve the intended result which has been decided from the start on time. I am not pretentious. I speak my mind and make it safe for others to do so. My collegues say about me that I am hardworking, joyful and problemsolving.</p>

<h3 id="toc_4">2022 04 24 Bioinf OlsAro Crop Biotech</h3>

<p>Expert in applied bioinformatics with 8+ years of experience in providing transparent and reproducible solutions for diverse areas of biological and medical research. Deep knowledge of underlying issues in need of bioinformatics support. Experience includes genetic epidemiology of rare diseases, integrative omics analysis of data from cancer tissue and immune cells, microarray analysis, preprocessing to final analysis of NGS output, sequence alignment and assembly working with non-human genomes from bacteria to fish species and wolf in evolutionary research and finally integrative analysis of proteomics, clinical and brain imaging data. Focused on keeping at the forefront of the latest within computational omics analysis. I seek to understand the healthcare environment and external trends. I have emotional intelligence, strong communication skills and am a great teammate. I am dynamic and work very hard to achieve the intended result which has been decided from the start on time. I am not pretentious. I speak my mind and make it safe for others to do so. My collegues say about me that I am hardworking, joyful and problemsolving. I love being creative and come up solutions that wasn’t thought of in advance but am also careful that no negative consequences have an impact on anyone else. </p>

<h3 id="toc_5">2022 04 28 Bioinformatiker Core Facilities GU</h3>

<p>Expert in applied bioinformatics with 8+ years of experience in providing transparent and reproducible solutions for diverse areas of biological and medical research. Deep knowledge of underlying issues in need of bioinformatics support. Experience includes genetic epidemiology of rare diseases, integrative omics analysis of data from cancer tissue and immune cells, microarray analysis, preprocessing to final analysis of NGS output. The integrative analysis, performed during phd, encompassed, gene expression, copy number variation, promoter methylation and histone occupancy data. Integrative analysis of proteomics, clinical and brain imaging data. Focused on keeping at the forefront of the latest within computational omics analysis. I seek to understand the healthcare environment and external trends. I have emotional intelligence, strong communication skills and am a great teammate. I am dynamic and work very hard to achieve the intended result which has been decided from the start on time. I am not pretentious. I speak my mind and make it safe for others to do so. My collegues say about me that I am hardworking, joyful and problemsolving. </p>

<h2 id="toc_6">2022 05 13 researchers in bioinformatics chalmers</h2>

<p>Expert in applied bioinformatics with 8+ years of experience in providing transparent and reproducible solutions for diverse areas of biological and medical research. Deep knowledge of underlying issues in need of bioinformatics support. Experience includes genetic epidemiology of rare diseases, integrative omics analysis of data from cancer tissue and immune cells, microarray analysis, preprocessing to final analysis of NGS output. The integrative analysis, performed during phd, encompassed, gene expression, copy number variation, promoter methylation and histone occupancy data. Integrative analysis of proteomics, clinical and brain imaging data. Focused on keeping at the forefront of the latest within computational omics analysis. I have emotional intelligence, strong communication skills and am a great teammate. I am dynamic and work very hard to achieve the intended result which has been decided from the start on time. I am not pretentious. I speak my mind and make it safe for others to do so. My collegues say about me that I am hardworking, joyful and problemsolving. </p>

<h2 id="toc_7">Objective</h2>

<h3 id="toc_8">2022 04 17 Sen Sci Mol Bio Bioinf AZ</h3>

<p>To use my scientific expertise to contribute to the efforts of the pharmaceutical industry towards the curation of disease. To take part in fulfilling the promises of genome editing.</p>

<h3 id="toc_9">2022 04 30 Clinical Application Consultant NGS themo fisher</h3>

<p>To contribute to improved biological/medical insight through the development of supporting computational infrastructure and algorithms</p>

<h3 id="toc_10">2022 04 24 Bioinf OlsAro Crop Biotech</h3>

<p>To further develop my skills in developing reproducible bioinformatics pipelines in a commercial setting and to contibute to the annotation of genomes and to the increased understanding of functional genomics.</p>

<h3 id="toc_11">2022 04 28 Bioinformatiker Core Facilities</h3>

<p>To further develop my skills in developing reproducible bioinformatics pipelines and contribute to improved insight into cnacer development and treatment through the development of supporting computational infrastructure and algorithms. To further develop my knowledge of and skills in applying AI and ML in piplines. To share attained knowldge.</p>

<h3 id="toc_12">2022  05 02 Senior Data Scientist Associate Director AZ R-135981-2</h3>

<p>To further develop my skills in developing reproducible bioinformatics pipelines and contribute to the efforts of the pharmaceutical industry towards the curation of disease and treatment. To further develop my knowledge of and skills in applying AI and ML in pipelines. To share attained knowledge.</p>

<h2 id="toc_13">Relevant competences</h2>

<h3 id="toc_14">General</h3>

<ul>
<li>NGS preprocessing and analysis </li>
<li>scientific communication and writing </li>
</ul>

<h3 id="toc_15">2022 04 17 Sen Sci Mol Bio Bioinf AZ</h3>

<ul>
<li>Using bioinformatic tools and databases to identify novel proteins based on available target class information – and novel DNA nucleases or DNA modifying enzymes for genome editing applications.</li>
<li>scientific communication and writing </li>
<li>project management </li>
</ul>

<h3 id="toc_16">2022 04 24 Bioinf OlsAro Crop Biotech</h3>

<ul>
<li>NGS preprocessing and analysis </li>
<li>R, git, Github, Nexflow, Docker</li>
<li>GWAS</li>
<li>scientific communication and writing </li>
<li>project management </li>
</ul>

<h3 id="toc_17">2022 04 28 Bioinformatiker Core Facilities GU</h3>

<ul>
<li>NGS preprocessing and analysis </li>
<li>shell, (Python), R, git, Github, Nexflow, Docker</li>
<li>Integrative cancer omics analysis</li>
<li>scientific communication and writing </li>
<li>project management </li>
</ul>

<h3 id="toc_18">2022  05 02 Senior Data Scientist Associate Director AZ R-135981-2</h3>

<ul>
<li>NGS preprocessing and analysis </li>
<li>shell, (Python), R, git, Github, Nexflow, Docker</li>
<li>Integrative cancer omics analysis</li>
<li>scientific communication and writing </li>
<li>project management </li>
</ul>

<h2 id="toc_19">Driving forces</h2>

<p>The increased understanding of life, who we are, and the curation of diseases</p>

<h2 id="toc_20">Strengths</h2>

<p>Dynamic and work very hard to achieve the intended result which has been decided from the start on time. I am not pretentious. I speak my mind and make it safe for others to do so.</p>

<h2 id="toc_21">Special competences</h2>

<p>Preprocessing of NGS output
:FASTQC, etc</p>

<p>Downstream analysis of NGS output
:Gene centric(DEG), GO and pathway centric, correlation module centric</p>

<p>Programming
:R, shell, Python</p>

<p>Version control
:git,Github</p>




</body>

</html>
